uds/transport_interface/can_transport_interface.py

```diff
@@ -457,7 +465,7 @@ def n_ar_measured(self) -> Optional[TimeMillisecondsAlias]:
         return self.__n_ar_measured
 
     @property  # noqa
-    def n_bs_measured(self) -> Optional[TimeMillisecondsAlias]:
+    def n_bs_measured(self) -> Optional[Tuple[TimeMillisecondsAlias, ...]]:
         """
         Get the last measured value of N_Bs time parameter.
```

Comment:
add reference to knowledge base N_Bs time explanation (same for other time parameters)